spring-cloud-kubernetes-fabric8-autoconfig/src/test/java/org/springframework/cloud/kubernetes/Fabric8NotInsideHealthIndicatorTest.java

```diff
@@ -65,6 +65,12 @@ public static void setUpBeforeClass() {
 	@AfterAll
 	public static void afterClass() {
 		server.after();
+		System.clearProperty(Config.KUBERNETES_MASTER_SYSTEM_PROPERTY);
```

Comment:
they are set, but not cleaned up. This does not cause any problems currently, but it enforces a good habit to clean-up